app/mailers/vacation_mailer.rb

```diff
@@ -0,0 +1,6 @@
+class VacationMailer < ApplicationMailer
+  def notify_vacation_request(vacation, admins_emails)
+    @vacation = vacation
+    mail(to: admins_emails, subject: "Punchclock - #{@vacation.user.name} is requesting vacations")
```

Comment:
Can the email subject also be fetched from I18n instead of hardcoded?

Reply:
> Can the email subject also be fetched from I18n instead of hardcoded?

Let me try

Reply:
> more so I think it should be requested vacation so it aligns with the pt-BR text

Cool, I'll improve that

Reply:
> Can the email subject also be fetched from I18n instead of hardcoded?

Sure, let me try

Reply:
> more so I think it should be `requested vacation` so it aligns with the pt-BR text

Cool, I'll improve that

Reply:
Improved that: https://github.com/Codeminer42/Punchclock/pull/294/commits/25ac9b417e8495baa060045a8b6e3844a4aee5dd ✅ 
thanks!